### DISCUSSION TITLE
Checking if there is an existing Personal Access Token

### DISCUSSION THREAD

**Discussion by berkayalan** [community/community#38242] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, I use Macbook with M1 chip. I try to check if there is an already existing ssh key in my local computer. When I check, I found this one : https://docs.github.com/en/authentication/connecting-to-github-with-ssh/checking-for-existing-ssh-keys

But when I wrote `ls -al ~/.ssh` to my terminal, I see these:

> total 8
> drwx------   3 berkayalan  staff    96 Nov  2 09:57 .
> drwxr-x---+ 40 berkayalan  staff  1280 Nov  4 08:18 ..
> -rw-r--r--   1 berkayalan  staff    92 Nov  2 09:57 known_hosts

What should I understand from here? Do I have 3 keys? I would be happy if you can help me. Thanks in advance!

**Comment by airtower-luna** [comment, marked as answer]
There is no private key in that directory listing. `.` is the current directory, `..` the parent directory, and `known_hosts` is a file in which SSH stores the public keys of servers you've approved.

So unless you have saved key pairs somewhere else, you don't have any.

Also keep in mind that an SSH key pair and a PAT are very different things. You'd need a PAT if you wanted to use Git over HTTPS instead of SSH.

**Reply by berkayalan** [reply]
Thanks for your answer. I was actually thinking the same.
